src/SprykerEco/Yves/FactFinderWebComponents/FactFinderWebComponentsFactory.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Yves\FactFinderWebComponents;
+
+use Spryker\Yves\Kernel\AbstractFactory;
+use SprykerEco\Yves\FactFinderWebComponents\Form\WebComponentConfigToTwigConfigTransformer;
+use Symfony\Component\Form\DataTransformerInterface;
+
+/**
+ * @method \SprykerEco\Yves\FactFinderWebComponents\FactFinderWebComponentsConfig getConfig()
+ */
+class FactFinderWebComponentsFactory extends AbstractFactory
+{
+    /**
+     * @return \SprykerEco\Yves\FactFinderWebComponents\WebComponentsConfigInterface
+     */
+    public function createWebComponentsConfig(): WebComponentsConfigInterface
```

Comment:
Please remove this method and use just `getConfig()`. Also remove this config interface.

Reply:
Because ->getConfig() is not available in Yves, I created and use WebComponentsConfigBuilder model instead.